pyscriptjs/tests/unit/pytitle.test.ts

```diff
@@ -25,16 +25,14 @@ describe("PyTitle", () => {
   })
 
   it("label renders correctly on the page and updates id", async () => {
-    instance.innerHTML = "Hello, world!"
-    // We need this to test mount_name works properly since connectedCallback
-    // doesn't automatically call checkId (should it?)
-    instance.checkId();
+    instance.innerHTML = "Hello, world!";
+    instance.id = "my-fancy-title";
 
     instance.connectedCallback();
 
     expect(instance.label).toBe("Hello, world!")
     // mount_name should be similar to: py_be025f4c_2150_7f2a_1a85_af92970c2a0e
```

Comment:
can remove this comment

Reply:
I mean `mount_name should be similar to: py_be025f4c_2150_7f2a_1a85_af92970c2a0e`

Reply:
thanks, killed it in 3eb07da